ui_framework/components/local_nav/_local_search.scss

```diff
@@ -53,6 +58,38 @@
 }
 
 /**
+ * 1. When it expands, the search input should lie on top of the content.
+ * 2. Expand as the user enters text.
+ * 3. Cap the height so that we can offer auto-completion suggestions below the input.
```

Comment:
i can expand the textarea as far as i want (multiple pages) .. the cap does not seem to be working